README.md

```diff
@@ -157,109 +175,115 @@ array(
+To load fixtures for the test, you need to define a simple ``YAML`` file in ``src/AppBundle/Tests/DataFixtures/ORM/books.yml``:
 
 ```yml
-    Lakion\ApiTestCase\Test\Entity\Product:
-        type: entity
-        table: test_product
-        id:
-            id:
-                type: integer
-                id: true
-                generator:
-                    strategy: AUTO
-        fields:
-            name:
-                type: string
-            price:
-                type: integer
+    Lakion\ApiTestCase\Test\Entity\Book:
+        book1:
+            name: "Lord of The Rings"
+            author: "J. R. R. Tolkien"
+        book2:
+            name: "Game of Thrones"
+            price: "George R. R. Martin"
 ```
 
-```yml
-    Lakion\ApiTestCase\Test\Entity\Product:
-        product1:
-            name: 'Phone'
-            price: 200
-        product2:
-            name: 'Book'
-            price: 15
-        product3:
-            name: 'Mug'
-            price: 5
+Finally, to use these fixtures in a test, just call a proper method:
+
+```php
+    public function testBooksIndexAction()
+    {
+        // This method require subpath to locate specific fixture file in your DataFixtures/ORM directory.
+        $this->loadFixturesFromFile('books.yml');  
+      
+        // There is another method that allows you to load fixtures from directory.
+        $this->loadFixturesFromDirectory('big_library');
+    }
 ```
 
-Ok so we got our data, now let's proceed with our testing case.
+Configuration Reference
+-----------------------
 
-```php
-//ProductControllerTest
+To customize your test suite configuration you can add a few more options to phpunit.xml:
 
-  public function testIndexAction()
-  {
-      // This method require subpath to locate specific fixture file in your DataFixtures/ORM directory.
-      $this->loadFixturesFromFile('product.yml');  
-      
-      // There is another method that allows you to load fixtures from directory.
-      $this->loadFixturesFromDirectory();
-      
-  // Ok you are ready to test responses.
-  
+```xml
+<php>
+    <server name="KERNEL_DIR" value="/path/to/dir/with/kernel" />
+    <server name="KERNEL_CLASS_PATH" value="/path/to/kernel/class" />
+    <server name="EXPECTED_RESPONSE_DIR" value="/path/to/expected/responses/" />
+    <server name="MOCKED_RESPONSE_DIR" value="/path/to/mocked/responses/" />
+    <server name="FIXTURES_DIR" value="/path/to/DataFixtures/ORM/" />
+    <server name="OPEN_ERROR_IN_BROWSER" value="true/false" />
+    <server name="OPEN_BROWSER_COMMAND" value="open %s" />
+    <server name="IS_DOCTRINE_ORM_SUPPORTED" value="true/false" />
+</php>
 ```
 
+ * `KERNEL_DIR` variable contains a path to kernel of your project. If not set, WebTestCase will look for AppKernel in the folder where you have your phpunit.xml file.
+ * `KERNEL_CLASS_PATH` allows you to specify exactly which class in which folder should be used in order to setup the Kernel. 
```

Comment:
It's worth mentioning, especially while having "`KERNEL_DIR` (...) If not set, WebTestCase will look for AppKernel in the folder where you have your phpunit.xml file." at the same time :)